crates/aptos-rosetta/src/construction.rs

```diff
@@ -410,11 +410,11 @@ async fn construction_parse(
                 (AccountAddress::ONE, APTOS_ACCOUNT_MODULE, CREATE_ACCOUNT_FUNCTION) => {
                     parse_create_account_operation(sender, &type_args, &args)?
                 }
-                (AccountAddress::ONE, STAKE_MODULE, SET_OPERATOR_FUNCTION) => {
-                    parse_set_operator_operation(sender, &type_args, &args)?
+                (AccountAddress::ONE, STAKING_PROXY_MODULE, SET_OPERATOR_FUNCTION) => {
```

Comment:
If we've switched to only staking_contract, we don't really need staking proxy contract in Rosetta anymore. You can just directly call the function in staking_contract::switch_operator_with_same_commission?